wasm/validator/src/default.rs

```diff
@@ -483,21 +457,40 @@ pub mod peer {
 pub mod domain {
     use super::*;
 
-    // TODO: We probably need a better way to allow accounts to modify domains.
-    tokens!(
-        pattern = {
-            #[derive(Token, ValidateGrantRevoke)]
+    declare_tokens! {
+        crate::default::domain::tokens::CanUnregisterDomain,
+        crate::default::domain::tokens::CanSetKeyValueInDomain,
+        crate::default::domain::tokens::CanRemoveKeyValueInDomain,
+    }
+
+    pub mod tokens {
+        // TODO: We probably need a better way to allow accounts to modify domains.
+        use super::*;
+
+        token! {
+            #[derive(ValidateGrantRevoke)]
             #[validate(permission::OnlyGenesis)]
-            pub struct _ {
-                pub domain_id: <Domain as Identifiable>::Id,
+            pub struct CanUnregisterDomain {
+                pub domain_id: DomainId,
             }
-        },
-        domain::tokens: [
-            CanUnregisterDomain,
-            CanSetKeyValueInDomain,
-            CanRemoveKeyValueInDomain,
-        ]
-    );
+        }
```

Comment:
So you've just downgraded `tokens!` macro to `token!` as it was before. This is how I did it initially. Maybe you're right, that's more readable

Reply:
yeah, I think it was unreadable